Firestore/Source/Remote/FSTDatastore.m

```diff
@@ -436,10 +438,56 @@ + (void)prepareHeadersForRPC:(GRPCCall *)rpc
 
 @end
 
+#pragma mark - GRXFilter
+
+/** Filter class that allows disabling of GRPC callbacks. */
+@interface GRXFilter : NSObject <GRXWriteable>
```

Comment:
The `GRX` namespace is reserved by gRPC. Putting this class in that namespace opens us up to having conflicting classes. This should be prefixed with `FST`. Maybe call it `FSTFilteredWriteable` or `FSTCallbackFilter`?

Reply:
I think the only name that makes sense here is "FSTCallbackFilter", since that leads to the member name of _callbackFilter.